### PR TITLE
⭐  Adding JotForm for Newsletter signup

### LINKED ISSUE #2287: 🐛Newsletters - Email validation
**Describe the bug**
Recently, there was an attempt to probe into our Power Automate flow, highlighting a lack of proper email validation measures in place.

**To Reproduce**
1. Go to https://www.ssw.com.au/newsletters
2. Fill all fields and email field without a valid email 
3. Submit the form

**Expected behavior**
It should not allow user to submit any invalid input

**Screenshots/Video**
e8b37ab5-5854-473b-a4f0-d7731cd28151

**Figure: PowerAutomate flow's payload for newsletter's subscription**

### LINKED ISSUE #2288: ✨ Newsletter - Create a JotForm for subscription
URL: https://www.ssw.com.au/newsletters

**Additional context**
As per my conversation with @wicksipedia , we need to create a Jotform to address the validation issues present in the existing newsletters subscription.

```[tasklist]
### Tasks
- [x] Create a JotForm
- [x] Embed it with our website
- [x] Create a Power Automate to submit subscription into CRM
- [x] ~~ZZ old flow that caters the submission of the newsletter~~ To be carried out on task https://github.com/SSWConsulting/SSW.Website/issues/2409
```

Screenshot:
image

**Figure: Newsletter's form**

### PR DESCRIPTION
**Pull request by andrewwaltosssw**
- Created InlineJotForm component
- Created and embedded newsletter subscription form using JotForm
- Updated locations previously showing old form to show new form
- Removed code for old newsletter form

- Affected routes: `/newsletters` `/*?liveStream=1`, `training/internship-fullstack`

- Fixed #2288

- [x] Include done video or screenshots
image
**Figure: Screenshot showing new form on live UG widget**

image
**Figure: Screenshot showing new form on newsletters page**

image
**Figure: Screenshot showing new form on training internship page**

### CODE REVIEW

File: components/liveStream/liveStreamWidget.tsx [hunk 1]
```diff
@@ -16,12 +16,14 @@ import { SpeakerInfo } from "../../services/server/events";
...
 import { SocialIcons } from "../socialIcons/socialIcons";
-import { SubNewsLettersForm } from "../subNewsletter/subNewsletterForm";
+import { InlineJotForm } from "../blocks";
 
 type LiveStreamWidgetProps = {
   isLive?: boolean;
 } & LiveStreamProps;
 
+const NEWSLETTER_SUBSCRIBE_JOTFORM_ID = "240991361342051";
+
```
Comment (amankumarrr) [inline components/liveStream/liveStreamWidget.tsx]: This should be coming from Tina and maybe have a centralized place where we will define the ID for the newsletters component.

**Review by amankumarrr — CHANGES_REQUESTED** [review]

File: components/liveStream/liveStreamWidget.tsx (hunk 1 above)
Comment (awaltos) [reply components/liveStream/liveStreamWidget.tsx]: Thanks Aman, i have moved this to Tina global index properties and referencing from there.

**Review by amankumarrr — APPROVED** [review]
LGTM

**Comment by andrewwaltosssw** [comment]
UI changes approved as per teams conversation with @hveraus

**Pull request merged**